### DISCUSSION TITLE
Security Onion vs VMware Carbon Black Response

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#12550] [General] [answered]
Hi All,
i am familiar with VMware Carbon Black Response and i have installed SO and i am trying to understand if SO have the same capabilities that CBR has and what i am interested in most is the details CBR holds regarding processes and the command line or executions they make which i find very useful.
i have uploaded a screenshot from CBR which indicates what information i can get from the agents and i would appreciate if someone can confirm what does exist in SO
![20240307_103338](https://github.com/Security-Onion-Solutions/securityonion/assets/110347919/b1b1d9db-f3d2-4b01-8910-8679b75eedf7)

Thanks,
Udi

**Comment by subs1138** [comment]
Are you using the Carbon Black Event Forwarder?  If so then the list is on the web page.
https://docs.elastic.co/en/integrations/carbonblack_edr

Alternatively your server should take anything and dump the info into a message field which you would then write GROK parsers for, but the CB Event Forwarder should make it easy.

**Comment by udi-mosh** [comment]
Hi,

I am not using the forwarder. I am looking to replace cb with so and i
wanted to know if the info i get with so elastic agent is equivalent to
what i get with cb as described in the thread.

בתאריך יום ב׳, 11 במרץ 2024, 21:09, מאת subs1138 ***@***.***

**Reply by dougburks** [reply, marked as answer]
You can see some of the features of Elastic Agent in the Elastic docs:
https://docs.elastic.co/en/integrations/endpoint

**Reply by udi-mosh** [reply]
looks promising...
